### PR TITLE
#13 - expand readme

### LINKED ISSUE #13: Expand readme with debian repo structure
Let's expand readme file with information about debian repository: how it should be structured, what metadata files we can work with and how to configure it.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #13 
Added info about debian repo into readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,65 @@ a fully-functionable Debian repository, which
 [`apt`](https://en.wikipedia.org/wiki/APT_(software)) 
 will perfectly understand.
 
+## Debian repository structure
+
+Debian repository [has](https://www.debian.org/doc/manuals/repository-howto/repository-howto#id442666)
```
Comment (olegmoz) [inline README.md:23]: @olenagerasimova first line of the document states that the document is obsolete, may we trust this source? Maybe there is updated documentation?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,65 @@ a fully-functionable Debian repository, which
...
+Size: 29238
+MD5sum: cee972bb5e9f9151239e146743a40c9c
+SHA1: d404261883ae7bd5a3e35abca99b1256ae070ed5
+SHA256: 421a5c6432cb7f9c398aa5c89676884e421ad0fde6bf13d0cadee178bf0daf7c
+Section: fonts
+Priority: extra
+Homepage: http://sourceforge.net/adobe/aglfn/
+Description: Adobe Glyph List For New Fonts
+ AGL (Adobe Glyph List) maps glyph names to Unicode values ...
+```
```
Comment (olegmoz) [inline README.md:78]: @olenagerasimova please state if this is example of Packages Indicies file or one of it's paragraphs (e.g. control file)

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see two comments

File: README.md (hunk 1 above)
Comment (olenagerasimova) [reply README.md:23]: @olegmoz this link is given here as this documentation explains files structure better than the new one and structure is still the same

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected and answered

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**